nemo_rl/algorithms/loss/__init__.py

```
@@ -0,0 +1,51 @@
+    "DistillationLossDataDict",
+    "DistillationLossFn",
+    "DPOLossConfig",
+    "DPOLossDataDict",
+    "DPOLossFn",
+    "NLLLossFn",
+    "PreferenceLossDataDict",
+    "PreferenceLossFn",
+    "SequencePackingLossWrapper",
+    "prepare_loss_input",
+    "wrap_loss_fn_with_input_preparation",
+]
```

Ruff (0.15.2)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)